cmds/coredhcp/config.yml.example

```diff
@@ -2,12 +2,14 @@ server6:
     interface: "eth0"
     #listen: 'your-unicast-address:547'
     plugins:
-        - server_id: LL 00:de:ad:be:ef:00
+        #- server_id: LL 00:de:ad:be:ef:00
```

Comment:
Are you changing this for any particular reason?

Reply:
No, it was a mistake.